public/documentation/embedding_crosswalk/crosswalk_aar.md

```diff
@@ -114,6 +114,8 @@ Then install it into the local Maven repository:
     }
     ```
 
+    Using mavenLocal() instead of remote repo url to refer local aar.
+
```

Comment:
So... what would you like?  Can I just change the sentence to:
 "Use <code>mavenLocal()</code> instead of the remote repo url to refer to a local aar."
Is there more we should do?